src/free.rs

```diff
@@ -128,17 +128,23 @@ pub fn zip<I, J>(i: I, j: J) -> Zip<I::IntoIter, J::IntoIter>
     i.into_iter().zip(j)
 }
 
-/// Create an iterator that first iterates `i` and then `j`.
+
+/// Takes two iterators and creates a new iterator over both in sequence. 
```

Comment:
Could you change "iterators" to "iterables" (seems to be in line with what we have elsewhere).

Reply:
Made the change